data/README.md

```diff
@@ -0,0 +1,5 @@
+# Data Sources for Tutorials
+
+| source                                        | format | reference                                                      | notes |
+| --------------------------------------------- | ------ | -------------------------------------------------------------- | ----- |
+| [Pre-Class Survey Results](surveyResults.csv) | csv    | results taken from google form sent out before semesters start |       |
```

Comment:
I switched the dataset to the quantitative results of our pre-class survey because (a) it is relevant to our students and (b) I couldn't find a good way to reference the `Time spent` one. 

I also thought in this `data/README` we can accumulate a series of simple and easy to work with datasets, each with a reference.